frontend/src/views/SyncDatabaseSchema/DatabaseSchemaSelector.vue

```diff
@@ -172,7 +172,7 @@ const hasSyncSchemaFeature = computed(() => {
 
 const shouldShowMoreVersionButton = computed(() => {
   return (
-    !hasSyncSchemaFeature.value &&
+    hasSyncSchemaFeature.value &&
```

Comment:
### Free plan

<img width="1396" alt="image" src="https://github.com/bytebase/bytebase/assets/24653555/7eeca4bd-843b-4b35-9392-3249cc49843c">


### Enterprise plan

<img width="1395" alt="image" src="https://github.com/bytebase/bytebase/assets/24653555/8dc6f4a3-c649-4653-81e3-3e0c2be5afd7">

---

I think we should show the more version button in free plan not enterprise plan. So here should still be `hasSyncSchemaFeature.value === false`?

Reply:
Looks like we can remove the show more button as it only used to remind the feature in the old UX (we won't show the history list in the past)

Reply:
SGTM

Reply:
Removed